spec/faraday_middleware/exceptional_response_spec.rb

```diff
@@ -0,0 +1,41 @@
+describe Mrkt::FaradayMiddleware::ExceptionalResponse do
+	before do
```

Comment:
Please only use spaces for the indentation.

```suggestion
  before do
```